Assets/LoggingManager/WriteToCSV.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+
+public class WriteToCSV : MonoBehaviour
+{
+    private string fileName;
+    private string savePath;
+    private string filePath;
+
+    public LogStore LogStore;
+
+
+    public WriteToCSV(LogStore logStore, string savePath, string filePrefix, string fileExtension)
+    {
+        this.fileName = filePrefix + "_" + logStore.Label + "_" +
+                        DateTime.Now.ToString("yyyy_MM_dd_HH_mm_ss_ffff") + fileExtension;
+        Init(logStore, savePath);
+    }
+
+    public WriteToCSV(LogStore logStore, string savePath, string fileName)
+    {
+        this.fileName = fileName;
+        Init(logStore, savePath);
+    }
+
+    public WriteToCSV(LogStore logStore, string filePath)
+    {
+        this.filePath = filePath;
+        this.LogStore = logStore;
+    }
+
+    private void Init(LogStore logStore, string path)
+    {
+        this.savePath = path == "" ? Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments) : path;
+        this.filePath = Path.Combine(savePath, fileName);
+        this.LogStore = logStore;
+    }
```

Comment:
The idea is to give the possibility to create the WriteToCSV object using :

- A full path
- A directory path and a file name
- A directory path, a file prefix and a file extension (in this case the file is named automatically with the date and the log label)

Each way of creating the object corresponds to a constructor.
Currently, only the third one is used, but I thought it would be a good idea to provide those features for later use.
The private Init() method is just here to avoid duplicate code when initializing the object.